plugins/source.go

```diff
@@ -154,25 +158,74 @@ func (p *SourcePlugin) Sync(ctx context.Context, logger zerolog.Logger, spec spe
 		}
 	}
 	wg.Wait()
-	logger.Info().Uint64("total_resources", summary.Resources).TimeDiff("duration", time.Now(), startTime).Msg("fetch finished")
+	logger.Info().Uint64("total_resources", summary.Resources).TimeDiff("duration", time.Now(), startTime).Msg("sync finished")
 	return &summary, nil
 }
 
-func (p *SourcePlugin) interpolateAllResources(tables []string) ([]string, error) {
-	if tables == nil {
-		return make([]string, 0), nil
+func (p *SourcePlugin) listAndValidateTables(tables, skipTables []string) ([]string, error) {
+	if len(tables) == 0 {
+		return nil, fmt.Errorf("list of tables is empty")
+	}
+
+	// return an error if skip tables contains a wildcard or glob pattern
+	for _, t := range skipTables {
+		if strings.Contains(t, "*") {
+			return nil, fmt.Errorf("glob matching in skipped table name %q is not currently supported", t)
```

Comment:
nit: I would put `is not supported` as we don't know if we want to support it in the future yet.

Reply:
updated 👍